main.cpp

```diff
@@ -38,18 +41,53 @@ int main(int argc, char *const argv[]) {
     }
 
     const IR::P4Program *program = nullptr;
-
+    // std::cout << "\nChecking " << options.file;
+    cstring command = "python3 /home/roborobo/projects/gauntlet_paper/gauntlet/"
+                      "validate_p4_translation.py -i ";
+    command += realpath(options.file, NULL);
+    // command += " > /dev/null";
+    // command += options.file;
+    // std::wcout << command;
+    int required_exit_code = system(command.c_str());
+    printf("%s\n", command);
+    std::cout << "Got code :" << required_exit_code << " for the main file";
```

Comment:
This should be debug prints instead std:cout or printf. I can add an example later so you can see what I mean.